src/java.base/share/classes/sun/security/ssl/CertificateRequest.java

```diff
@@ -370,12 +382,19 @@ public void consume(ConnectionContext context,
 
             X509ExtendedKeyManager km = chc.sslContext.getX509KeyManager();
             String clientAlias = null;
-            if (chc.conContext.transport instanceof SSLSocketImpl) {
-                clientAlias = km.chooseClientAlias(crm.getKeyTypes(),
-                    crm.getAuthorities(), (SSLSocket)chc.conContext.transport);
-            } else if (chc.conContext.transport instanceof SSLEngineImpl) {
-                clientAlias = km.chooseEngineClientAlias(crm.getKeyTypes(),
-                    crm.getAuthorities(), (SSLEngine)chc.conContext.transport);
+
```

Comment:
Many of these lines are > 80 chars, which makes side-by-side challenging.  Please keep to a max of 80.

Reply:
Slacked with Kevin and pointed out locations.

Reply:
Slacked with Kevin and resolved my concern.